.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,23 +7,19 @@ linked together.
 
 ### Checklist
 
-- [ ] Did you run `npm test`?
+- [ ] Have you added unit and functional tests as appropriate?
 - [ ] Did you update/add documentation for new methods or changed functionality?
-
-Finally, is it ready to be reviewed and merged: yes :white_check_mark: / no :red_circle:
+- [ ] Have you merged the target branch down into this one?
```

Comment:
When testing locally, we should be testing a version which includes all changes in the target branch. Otherwise, we might be missing incompatible commits between the branches.